solidity/contracts/fully-backed/FullyBackedECDSAKeepFactory.sol

```diff
@@ -0,0 +1,254 @@
+/**
+▓▓▌ ▓▓ ▐▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓    ▓▓▓▓▓▓▓▀    ▐▓▓▓▓▓▓    ▐▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▄▄▓▓▓▓▓▓▓▀      ▐▓▓▓▓▓▓▄▄▄▄         ▓▓▓▓▓▓▄▄▄▄         ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▓▓▓▓▓▓▓▀        ▐▓▓▓▓▓▓▓▓▓▓         ▓▓▓▓▓▓▓▓▓▓▌        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓▀▀▓▓▓▓▓▓▄       ▐▓▓▓▓▓▓▀▀▀▀         ▓▓▓▓▓▓▀▀▀▀         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▀
+  ▓▓▓▓▓▓   ▀▓▓▓▓▓▓▄     ▐▓▓▓▓▓▓     ▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌
+▓▓▓▓▓▓▓▓▓▓ █▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+
+                           Trust math, not hardware.
+*/
+
+pragma solidity 0.5.17;
+
+import "./FullyBackedECDSAKeep.sol";
+
+import "./FullyBackedBonding.sol";
+import "../api/IBondedECDSAKeepFactory.sol";
+import "../KeepCreator.sol";
+import "../GroupSelectionSeed.sol";
+import "../CandidatesPools.sol";
+
+import {
+    AuthorityDelegator
+} from "@keep-network/keep-core/contracts/Authorizations.sol";
+
+import "@keep-network/sortition-pools/contracts/api/IFullyBackedBonding.sol";
+import "@keep-network/sortition-pools/contracts/FullyBackedSortitionPoolFactory.sol";
+import "@keep-network/sortition-pools/contracts/FullyBackedSortitionPool.sol";
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/// @title Fully Backed Bonded ECDSA Keep Factory
+/// @notice Contract creating bonded ECDSA keeps that are fully backed by ETH.
+/// @dev We avoid redeployment of bonded ECDSA keep contract by using the clone factory.
+/// Proxy delegates calls to sortition pool and therefore does not affect contract's
+/// state. This means that we only need to deploy the bonded ECDSA keep contract
+/// once. The factory provides clean state for every new bonded ECDSA keep clone.
+contract FullyBackedECDSAKeepFactory is
+    IBondedECDSAKeepFactory,
+    KeepCreator,
+    AuthorityDelegator,
+    GroupSelectionSeed,
+    CandidatesPools
+{
+    FullyBackedSortitionPoolFactory sortitionPoolFactory;
+    FullyBackedBonding bonding;
+
+    using SafeMath for uint256;
+
+    // Sortition pool is created with a minimum bond of 20 ETH to avoid
+    // small operators joining and griefing future selections before the
+    // minimum bond is set to the right value by the application.
+    //
+    // Anyone can create a sortition pool for an application with the default
+    // minimum bond value but the application can change this value later, at
+    // any point.
+    uint256 public constant minimumBond = 20e18; // 20 ETH // TODO: Decide on value
```

Comment:
Let's use the same value as in `BondedECDSAKeepFactory`. What's proposed here is fine.